README.md

```diff
@@ -14,31 +15,31 @@ Microgateway offers you a proxy that is capable of performing security validatio
 
 #### Architecture
 
-The following diagram illustrates the process of getting an API (or a selected set of APIs) to be hosted on a Microgateway.
+The following diagram illustrates the process of exposing the micro services via Microgateway using Open API defintion.
 
-![Alt text](architecture.png?raw=true "Title")
+![Alt text](architecture-new.png?raw=true "Title")
+
+###### Dev Phase
+
+* API developer creates a microgateway project using a microgateway controller(toolkit)
+* Adds the open API definitions of microservices into the project
+* Developer defines endpoints and interceptors for the api/resources using the definition.yaml inside the project
+* Builds the project and generates executables, images and k8s artifacts
 
 #### Running the microgateway
 
 Running the Microgateway is a 3 step process. The first two steps are involved in building the runtime.
 
- 1. Setting up a microgateway project.
+ 1. initiating a microgateway project.
  1. Building the microgateway project and creating a microgateway distribution.
  1. Running the microgateway distribution.
 
-##### Setting up a microgateway project
-
-To setup a microgateway project, a developer can choose two ways.
-
- 1. Create a microgateway project for a single API
- 1. Create a microgateway project for a group of APIs
+##### Initiating  a microgateway project
 
-The first step of setting up a microgateway project includes connecting to the API Publisher (Management Layer) and downloading the relevant API artifacts (JSON representation of the APIs). Once this step is completed it will convert the JSON representation of the APIs to Ballarina source code. The annotations that go into these source files (k8s, docker annotations, etc) are governed by a config file which the microgateway can see. These generated files can optionally be managed via source management repositories (Git).
+Once the project is initiated the default folder structure will be created for the respective folder.
+Empty api_definitions folder and definitions.yaml will be created inside the folder.API developer can add multiple open API definitions inside the
+api_definitions file and define endpoints and interceptors for the resources in definitions.yaml
```

Comment:
```suggestion
`api_definitions` file and define endpoints and interceptors for the resources in `definitions.yaml`
```

Reply:
There is already a sub heading called "How to run the microgateway distribution"